parallel_hashmap/btree.h

```diff
@@ -712,7 +712,7 @@ namespace phmap {
 
         template <
             typename Compare, typename K, typename LK,
-            phmap::enable_if_t<!std::is_same<bool, phmap::result_of_t<
+            phmap::enable_if_t<!std::is_same<bool, phmap::invoke_result_t<
                                                        Compare(const K &, const LK &)>>::value,
```

Comment:
Compare, const K &, const LK &>>::value,

Reply:
I had it right at one point, but I put myself into some kind of funk.  I'm still working on my git skills.